tests/unit/commands/insert-new-li-command-test.ts

```diff
@@ -113,6 +113,7 @@ module('Unit | commands | insert-new-li-command-test', (hooks) => {
             <text>abc</text>
           </li>
           <li>
+            <text>${INVISIBLE_SPACE}</text>
```

Comment:
all bow to the mighty invisible space. I think we might want to write an RFC on how to do this properly...

Reply:
yeah... its annoying
the easiest place to clean up is on write, but I really want to avoid making changes to the vdom while writing

Reply:
we may not need to have the invisible space in the vdom at all is what I'm thinking... it's only relevant when writing a selection (I think). anyway not a discussion we should have on this PR :).

Reply:
ooh yeah good idea